server/db/_db.js

```diff
@@ -4,6 +4,6 @@ var path = require('path');
 var Sequelize = require('sequelize');
```

Comment:
I think the logging is annoying, but if you guys like to read it, then fuck you.